BEP2.md

```diff
@@ -0,0 +1,212 @@
+      - [4.3.5 Mint Tokens](#435-mint-tokens)
+      - [4.3.6 Burn Tokens](#436-burn-tokens)
+  * [5. License](#5-license)
+
+
+
+
+## 1.  Summary
+
+This BEP describes a proposal for token management on the Binance Chain.
+
+## 2.  Abstract
+
+BEP-2 Proposal describes a common set of rules for token management within the Binance Chain ecosystem. It introduces the following details of a token on Binance Chain:
+
+- What information makes a token on Binance Chain
+- What actions can be performed on a token on Binance Chain
+
+## 3.  Status
+
+This BEP is under implementation. 
+
+## 4.  Motivation
+
+Design and issue asset on the Binance Chain, as the basic economic foundations of the blockchain.
+
+## 5.  Specification
+
+### 5.1  Native Token on Binance Chain: BNB
+
+The Binance Token, BNB, is the native asset on Binance Chain and created within Genesis Block. As the native asset, BNB would be used for fees (gas) and staking on Binance Chain. BNB was an ERC20 token, but after Binance Chain is live, all BNB ERC20 tokens are swapped for BNB token on Binance Chain. All users who hold BNB ERC20 tokens can deposit them to Binance.com, and upon withdrawal, the new Binance Chain native tokens will be sent to their new addresses.
+
+### 5.2  Token Properties
+
+- Source Address: Source Address is the owner of the issued token.
+
+- Token Name: Token Name represents the long name of the token - e.g. "MyToken". 
+
+- Symbol: Symbol is the identifier of the newly issued token.
+
+- Total Supply: Total supply will be the total number of issued tokens. 
+
+- Mintable: Mintable means whether this token can be minted in the future, which would increase the total supply of the token
+
+###  5.3 Token Management Operation
+
+#### 5.3.1  Issue token
+
+Issuing token is to create a new token on Binance Chain. The new token represents ownership of something new, and can also peg to existing tokens from any other blockchains.
+
+**Data Structure for Issue Operation**: A data structure is needed to represent the new token:
+
+| **Field**    | **Type** | **Description**                                              |
+| :------------ | :-------- | :------------------------------------------------------------ |
+| Name         | string   | Name of the newly issued asset, limited to 32 unicode characters,  e.g. "ABCcoin" |
+| Symbol       | string   | The length of the string for representing this asset is between 3 and 8 alphanumeric characters and is case insensitive. ".B" suffixed symbol is also allowed for pegging to those tokens already exist on other chains. The symbol is suffixed with the first 3 bytes of the issue transaction hash to remove a constraint of requiring unique token names. The native token, BNB, does not require this suffix. |
+| Total Supply | int64    | The total supply for this token can have a maximum of 8 digits of decimal and is boosted by 1e8 in order to store as int64. The amount before boosting should not exceed 90 billion. |
```

Comment:
Why is there an artificial supply cap?

Reply:
Ah, the cap is similar to the amount possible when subdividing by 1e8. The real limit is ~92.23372 Billion.